config/performance.txt

```diff
@@ -0,0 +1,7 @@
+
+## Set this to disable parallax updates after this amount of players have joined the server
+## This will cause the parallax to be a static background.
+# PARALLAX_DISABLE_POP 70
+## The population at which ghosts should stop seeing parallax updates.
+## Alive players will still receieve them.
+# PARALLAX_GHOST_DISABLE_POP 50
```

Comment:
Are these the recommended production values?

Reply:
They could probably be lowered tbh.
I should elaborate on the comment to say that parallax still exists, it just is static (which is barely noticable in game)

Reply:
Ghost pop should be 40 actually